pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -170,7 +172,7 @@ func (vm *windows) ConfigureHybridOverlay(nodeName string) error {
 	// Start the hybrid-overlay in the background over ssh.
 	// TODO: This will be removed in https://issues.redhat.com/browse/WINC-353
 	go vm.Run(remoteDir+wkl.HybridOverlayName+" --node "+nodeName+
-		" --k8s-kubeconfig c:\\k\\kubeconfig > "+logDir+"hybrid-overlay.log 2>&1", false)
+		" --k8s-kubeconfig c:\\k\\kubeconfig > "+hybridOverlayLogDir+"hybrid-overlay.log 2>&1", false)
```

Comment:
Given you are changing this, how about using `--logfile` option instead of redirecting?